### ISSUE TITLE
Link text is not displayed

### ISSUE THREAD

**Issue by selivan** [algorys/redissue#4]
Dokuwiki version: 2015-08-10a "Detritus"
Dokuwiki template: dokuwiki (standard template)
Redissue version: current master (commit f13acd67283d8eadaaced8f3d5bc9cee4559a690 )

Wiki markup: `<redissue id='#482' text="ZZZ" />`

Result in page source:

```
<p>
<img src="lib/plugins/redissue/images/redmine.png" class="redissue"/> <a href="http://redmine.unitpay.ru/issues/482" class="redissue redissue-status-open">[#482]Сделать интеграцию redmine и dokuwiki</a> <span class="status">new</span> <img src="lib/plugins/redissue/images/open.png"/>
</p>
```

Text "ZZZ" does not appear anywhere.

**Comment by algorys** [comment]
Hello,

Normally your text "ZZZ" appears only if user who's visiting page have no access right to the project (and issue) in Redmine.

Have you tried to connect to Dokuwiki with a Redmine user who can't see this project ?

**Comment by selivan** [comment]
My bad, missed this in documentation. Thanks for help.

BTW, it may be useful to have ability to override issue name, sometimes they are long and you need only short reference like #NNN.

**Comment by algorys** [comment]
Yes that's true.

I'll look for that.

**Comment by algorys** [comment]
@selivan that's available now :simple_smile: see #7
